docs/architecture/interconnect.rst

```diff
@@ -0,0 +1,26 @@
+Interconnect PIPs
+=================
+
+Fake PIPs
+---------
+
+Some PIPs are not "real", in the sense that no bit pattern in the bit-stream correspond to the PIP being used. This is the case for all the PIPs in the switchbox in a CLB tile (ex: CLBLM_L_INTER): They either correspond to buffers that are always on (i.e. 1:1 connections such as `CLBLL_L.CLBLL_L_AQ->CLBLL_LOGIC_OUTS0`), or they correspond to permutations of LUT input signals, which is handled by changing the LUT init value accordingly, or they are used to "connect" two signals that are driven by the same signal from within the CLB.
+
+The bit switchbox in an INT tile also contains a few 1:1 connections that are in fact always present and have no corresponding configuration bits.
+
+Regular PIPs
+------------
+
+Regular PIPs correspond to a bit pattern that is present in the bit stream when the PIP is used in the current design. There is a block of up to 10-ish bits for each destination signal. For each configuration (i.e. source net that can drive the destination) there is a pair of bits that is set.
```

Comment:
It's unclear if this sentence is generally true for PIPs outside of the INT switch box.

Reply:
Added FIXME.